dao-programs/programs/tutorial/src/lib.rs

```diff
@@ -2,12 +2,24 @@ use anchor_lang::prelude::*;
 
 mod constants;
 mod errors;
-pub mod state;
 pub mod events;
+pub mod state;
 
 pub mod instructions;
 use instructions::*;
 
+use solana_security_txt::security_txt;
+
+security_txt! {
```

Comment:
After reading the [docs on security-txt](https://github.com/neodyme-labs/solana-security-txt) it's clear that this macro will generate a string inside of the start and end markers (the public constant `security_txt`).

It looks like there's a small formatting glitch in [the security-txt parser source](https://github.com/neodyme-labs/solana-security-txt/blob/fa964e9ea227d806e0c9518e309210f09702823a/security-txt/src/parser.rs#L99), just a missing newline.

Other than that, just a couple of questions 😀

- Why not use `https://builderdao.io` as the contact site? It redirects to the Notion site for now, but it would be one less change to make in the future.
- Why link to the Solana Labs `SECURITY.md`? It seems like we should have our own policy linked here, otherwise this could be confusing for the intended audience.
- Given the simplicity of the implementation, couldn't we just add it to the program directly, [instead of using the dependency](https://github.com/neodyme-labs/solana-security-txt#security-of-this-crate)?

Otherwise, LGTM 👍

Reply:
> Why link to the Solana Labs SECURITY.md?


This PR is about integrating the `security-txt` feature. Defining a Security policy is another task on this own, which why for the alpha-release we will default to the Security of Solana core team, for the sake of demonstration. 


> couldn't we just add it to the program directly..?


Better to rely on the dependency as:
* Our goal is not to maintain our own version of this kind of feature, better to delegate this to a dedicated team.
* The parser allow us to not add any fancy field and to respect `the standard` 

> Given the simplicity of the implementation


Thanks, senior code is by definition; code easy to read, to understand, to maintained and which add useful feature. The opposite of junior code which is hard to read to understand, nearly impossible to maintain and which add questionable feature, imho.